packages/commonwealth/client/scripts/navigation/CommonDomainRoutes.tsx

```diff
@@ -151,6 +153,11 @@ const CommonDomainRoutes = ({
     path="/createCommunity"
     element={withLayout(CreateCommunityPage, { type: 'common' })}
   />,
+  <Route
+    key="/unSubscribe/:userId"
+    path="/unSubscribe/:userId"
```

Comment:
prob just use lowercase for `unSubscribe`

Reply:
in other routes they are using the camelCase  @mzparacha